container/mk-docker.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs, contents }:
+
+with pkgs;
```

Comment:
file-level `with` is considered rude similar to `from ... import *` in Python

Reply:
open to suggested improvements that don't unnecessarily increase verbosity

Reply:
In this specific case it would actually decrease verbosity

Reply:
ha, so it would